Source/SIMPLib/REST/V1Controllers/ExecutePipelineController.h

```diff
@@ -0,0 +1,91 @@
+/* ============================================================================
+ * Copyright (c) 2017 BlueQuartz Softwae, LLC
```

Comment:
Randomly noticed that Software is misspelled.  Try a mass search and replace as this does not seem to be the only one.  You might as well update the years at the same time.  These files were hit at least in 2018, and fixing the company name in the copyright puts it in 2019.

Reply:
Done